docs/generate.sh

```diff
@@ -2,12 +2,16 @@
 
 set -e
 
-FOLDER=docs/source/demos/
+FOLDER=docs/source/demos
 
 for file in "${FOLDER}/01_linear_function" "${FOLDER}/02_multilinear_function" "${FOLDER}/03_nonlinear_function"; do
 	jupytext --to ipynb "${file}.py"
     jupyter-nbconvert --to markdown --execute --ExecutePreprocessor.timeout=None "${file}.ipynb"
 done
 
+FOLDER=docs/source/jaxducc
```

Comment:
* Why redefine `FOLDER` here? If we want to redefine it, I would make it a lower-case variable 
* Isn't the true folder rather "docs/source/contrib"?